relay-server/src/metrics_extraction/transactions.rs

```diff
@@ -499,6 +507,16 @@ fn extract_transaction_metrics_inner(
         tags_with_satisfaction.clone(),
     ));
 
+    // Count the transaction towards the root
+    sampling_metrics.push(Metric::new_mri(
+        METRIC_NAMESPACE,
```

Comment:
we need to think about internal metric?

Reply:
I think I've added this comment during our discussion, so are you ok with NAMESPACE? If yes we can resolve it.